spec/controllers/tag_set_nominations_controller_spec.rb

```diff
@@ -906,8 +906,8 @@ def add_freeform_nominations(num)
             fake_login_known_user(random_user)
           end
 
-          it 'redirects and returns an error message' do
-            put :update, id: tag_set_nomination.id, tag_set_id: owned_tag_set.id, tag_set_nomination: {}
+          it 'redirects and returns an error message', focus: true do
```

Comment:
Will you need this later? If so, no worries! If not, we don't really filter our tests, so could you remove it?